addons/nvidia/0.3.x/nvidia-2.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: nvidia
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: nvidia
+    kubeaddons.mesosphere.io/provides: nvidia
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.0-2"
```

Comment:
The revision number should reference the app number. What is this semver referencing?

Reply:
Updated to 0.2.0-2 and rebased to master

Reply:
What does this mean, @shaneutt ? The last one was 0.3.0-1 should this not be 0.3.0-2?

btw, the links are broken: https://github.com/mesosphere/kubernetes-base-addons#overview